holopy/scattering/theory/mielensfunctions.py

```diff
@@ -0,0 +1,477 @@
+import numpy as np
+from scipy.special import j0, j1, spherical_jn, spherical_yn
+from scipy import interpolate
+
+
+NPTS = 100
+LEGGAUSS_PTS_WTS_NPTS = np.polynomial.legendre.leggauss(NPTS)
+
+
+# TODO:
+# fast integration of oscillatory functions.
+
+
+class MieLensCalculator(object):
+    def __init__(self, particle_kz=10.0, index_ratio=1.1, size_parameter=10.0,
```

Comment:
why are there default values for particle_kz, index_ratio, and size_parameter? Are these just for testing purposes? It might be beneficial to have it throw an exception if a user forgets/fails to specify one, rather than using an arbitrary value.

Reply:
This should be fixed with commits 4355790 and 1d6f577